### PR TITLE
Developer improvement

### PR DESCRIPTION
**Pull request by posix4e**
I wanted to get involved in Signal Server and I thought I could clean up the developer experience get started. Hopefully the // FREEBIE tag is how you signify that the money should be kept in the foundation. Happy if all, some or none of this is welcome.

### CODE REVIEW

**Comment by posix4e** [comment]
I also attempted to get some of the changes in https://github.com/WhisperSystems/Signal-Server/pull/17 in there. Hopefully we can close that old pr now

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: java
+jdk:
+    - oraclejdk8
```
Comment (moxie0) [inline .travis.yml]: Please no travis, thanks

File: bin/test [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -exu
+docker run -it --rm \
+       -v ~/.m2:/var/maven/.m2 \
+       -v "$PWD":/usr/src/mymaven -w /usr/src/mymaven \
+       -u 1000 \
+       -e MAVEN_CONFIG=/var/maven/.m2 \
+       maven:alpine \
+       mvn -Duser.home=/var/maven clean test
```
Comment (moxie0) [inline bin/test]: I'd prefer not to mix docker (or shell scripts!) into this

**Comment by posix4e** [comment]
No problem.

**Comment by posix4e** [comment]
https://github.com/WhisperSystems/Signal-Server/pull/80

**Pull request closed without merging**